### PR TITLE
Use {{page.version}} for links to ref pages.

### PR DESCRIPTION
**Pull request by steveperry-53**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4472)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@pwittrock, Do you think this strategy will work going forward?
Instead of making links like this:
* /docs/api-reference/v1.7/#...
Make links like this:
* /docs/api-reference/{{page.version}}/#...

**Comment by ahmetb** [comment]
I recommend adding this to travis.yml so such files cannot be merged in the first place.

    test -z "$(grep -E -Iirn --color "/docs/api-reference/v\d\.\d+" --include \*.md --exclude ./docs/reference/index.md . | tee /dev/stderr)"

/lgtm

File: .travis.yml [hunk 1]
```diff
@@ -27,3 +27,5 @@ script:
 - $GOPATH/bin/md-check --root-dir=$HOME/gopath/src/k8s.io/kubernetes.github.io
 - ./verify-docs-format.sh
 - $GOPATH/bin/mungedocs --verbose --verify --upstream=origin --root-dir=$HOME/gopath/src/k8s.io/kubernetes.github.io/docs/ --repo-root=$HOME/gopath/src/k8s.io/kubernetes.github.io --skip-munges=remove-whitespace,blank-lines-surround-preformatted,header-lines,sync-examples,analytics,analytics,kubectl-dash-f,table-of-contents,md-links,kubectl-dash-f
+- test -z "$(grep -EIirn "/docs/api-reference/v[0-9]\.[0-9]{1,2}" --include \*.md --exclude ./docs/reference/index.md ." --include \*.md --exclude ./docs/reference/index.md . | tee /dev/stderr)"
+- test -z "$(grep -EIirn "/docs/resources-reference/v[0-9]\.[0-9]{1,2}" --include \*.md --exclude ./docs/reference/index.md ." --include \*.md --exclude ./docs/reference/index.md . | tee /dev/stderr)"
```
Comment (ahmetb) [inline .travis.yml]: you have `--include \*.md --exclude ./docs/reference/index.md .`  typed twice.

**Comment by steveperry-53** [comment]
@ahmetb, Let's save the additions to travis.yml for another PR. I tried to to add the test you suggested, but I can't get it to work. I can't get it to exclude docs/reference/index.md.

**Pull request closed without merging**